devfile.yaml

```diff
@@ -75,8 +60,8 @@ commands:
         type: exec
         command: |
           set -o xtrace;
-          rm -rf /usr/local/apache2/htdocs/v3/*;
-          cp -rf /projects/build/v3/* /usr/local/apache2/htdocs/v3
+          rm -rf /usr/local/apache2/htdocs/v3;
+          cp -rf ${CHE_PROJECTS_ROOT}/che-plugin-registry/output/v3 /usr/local/apache2/htdocs/
         component: eclipse-che-plugin-registry
 
   - name: 3. publish to surge.sh
```

Comment:
`workdir` should be `${CHE_PROJECTS_ROOT}/che-plugin-registry/output`

Reply:
fixed